<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="file_path">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="file_content"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="get_file">
        <property name="text">
         <string>提取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="do_lex">
        <property name="text">
         <string>词法分析</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="do_down">
        <property name="text">
         <string>语法分析(递归下降)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="do_LL1">
        <property name="text">
         <string>语法分析(LL1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="do_sem">
        <property name="text">
         <string>语义分析</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
